src/index.js

```diff
@@ -10,45 +11,154 @@ class Todo {
   }
 }
 
-let todos = [
-  new Todo('Hello world 1', false, 0),
-  new Todo('Hello world 2', false, 1),
-  new Todo('Hello world 5', false, 4),
-  new Todo('Hello world 3', false, 2),
-  new Todo('Hello world 4', false, 3),
-];
+let todos = [];
 
-function populate() {
-  todos.sort((a, b) => (a.index > b.index ? 1 : -1));
-  todos.forEach((todo) => {
-    const li = document.createElement('li');
-    li.innerHTML = `
-    <div class="flex">
+function createTodoElement(todo) {
+  const li = document.createElement('li');
+  li.innerHTML = `
+    <div class="flex todo-element">
       <div>
           <input type="checkbox" class="checkbox" 
           ${todo.completed ? 'checked' : ''}>
           <span>${todo.description}</span>
       </div>
-      <span class="material-icons">
+      <span class="material-icons edit-icon" style="cursor: pointer">
           more_vert
       </span>
     </div>
     <hr>`;
 
-    button.parentElement.insertBefore(li, button);
+  return li;
+}
+
+function createReplaceTodoElement(todo) {
+  const html = `
+  <div>
+    <input type="checkbox" class="checkbox" 
+    ${todo.completed ? 'checked' : ''}>
+    <span>${todo.description}</span>
+  </div>
+  <span class="material-icons edit-icon" style=" cursor: pointer">
+      more_vert
+  </span>
+    `;
+
+  return html;
+}
+
+function addTodoElement(todo) {
+  const li = createTodoElement(todo);
+  button.parentElement.insertBefore(li, button);
+}
+
+function populate() {
+  todos.sort((a, b) => (a.index > b.index ? 1 : -1));
+  todos.forEach((todo) => {
+    addTodoElement(todo);
   });
 }
```

Comment:
- [ ] It would be better if the index of the todo is the index of the length of the array. As one of the project requirements. Please fix this minor issue. 

![Capture](https://user-images.githubusercontent.com/64173189/134588757-7ef257b8-4777-45ca-bce3-885c828bd6a8.JPG)

Currently the index start from `0` so just instead of starting from `0` started from `1`

🛠️ 👍